src/content/docs/en/welcome.mdx

```diff
@@ -0,0 +1,83 @@
+
+## General Philosophy
+
+Ultramarine Linux is a spiritual successor to Korora Linux, intending to make an operating system that "just works" for people of all levels of experience.
+
+To do this, we stick to a set of values:
+
+### Ultramarine is Pragmatic.
+
+Where Fedora tries to adhere to ideology (that’s perfectly fine), by shipping an (almost) entirely free system, Ultramarine works off the model of least resistance. That means we try to make it easy for the user, even if we have to ship some proprietary drivers and codecs that Fedora wouldn’t be comfy with out-of-box.
+
+### Ultramarine is Innovative.
+
+We believe that recent innovations in the Linux space serve to greatly improve the user and developer experience. Technologies like Flatpak and xdg-portals allow developers to provide great experiences to our users more consistently. Work done upstream improves the experience for our users, and work done in the Ultramarine ecosystem; like accent colours, and Chromebook support, improves the larger Linux ecosystem. It's a win-win.
+
+### Ultramarine is User-friendly.
+
+While it may seem trivial to more advanced users to tweak a few settings and install some packages from 3rd party repositories, it may take people who are new to Linux (or even just new to Fedora) hours to figure out what packages they need, and where to find them. Ultramarine allows users new and old to get up and running quickly.
+
+## Planned Features
+
+- Welcome app for user onboarding with quick access to common actions
+- System configuration application similar to YaST
+- Custom installer - Easier to use compared to Anaconda which has a overwhelming interface for new users
+
+## FAQ
+
+### What does the migration script do?
+
+The migration scripts install RPMFusion, Terra repository, and our signing keys, then installs the bare minimum of an Ultramarine Linux installation on top of an existing Fedora installation (Swapping Fedora release packages with the Ultramarine ones).
+
+The script only installs the bare minimum required for Ultramarine Linux to work because we don't want to add additional bloat to your current system.
+
+After you're migrated, you'll get the latest and greatest Ultramarine features as we build them!
+
+### Why does this distro exist? What difference does it have other than just Fedora with RPMFusion?
+
+Ultramarine Linux was created out of frustration with the legal limits of Fedora. As Fedora (and Red Hat) is an American entity, there are legal restrictions on what software can be
+packaged in the distribution due to the US patent system.
+
+Ultramarine, on the other hand, is based in Thailand and is not subject to any of the copyright restrictions on what software can be packaged.
+
+The Ultramarine team aims to make Fedora a little more user-friendly by allowing users to install or package any software they want as long as someone maintains it in Flathub, RPMFusion, or Terra.
+

```

Comment:
Mention more than just allowing you to have any software.. like for example, UX improvements, custom apps, etc...